app/response_processor.py

```diff
@@ -199,6 +199,10 @@ def remote_call(self, request_url, json=None, data=None, headers=None,
 
         except MaxRetryError:
             self.logger.error("Max retries exceeded (5)", request_url=request_url)
+            raise RetryableError
```

Comment:
Is this right? If you've hit the maximum number of retrys then why would you raise it as something retryable? or have I missed something here?

Reply:
The max retries is at the urllib level . It will have retried 5 times to get here . The Retryable error will then result in a NAK back to rabbit and end up retrying the entire message some time in the future . In the meantime we will have generated an error that splunk should have alerted on . Prior to this the exception bubbled past the log as it was a ConnectionError, the app half fell over and no splunk error raised